config/billing_rates.yml

```diff
@@ -12,7 +12,6 @@
 - { id: d31188b8-ced8-4bab-bb73-ee6b0c6a4806, resource_type: VmCores,                 resource_family: standard,            location: hetzner-hel1,   unit_price: 0.0005740000, billed_by: duration, active_from: 2024-08-22T00:00:00Z }
 - { id: a63fc107-176f-41e6-baf6-a1399446f616, resource_type: VmCores,                 resource_family: standard,            location: github-runners, unit_price: 0.0005257320, billed_by: duration, active_from: 2024-05-27T00:00:00Z }
 - { id: 724069e2-fa28-46d9-918c-666d1fbe80ba, resource_type: VmCores,                 resource_family: standard,            location: leaseweb-wdc02, unit_price: 0.0008000992, billed_by: duration, active_from: 2024-10-07T00:00:00Z }
-- { id: 534e85b6-2ba2-44c7-b799-8a1b99f8cd50, resource_type: VmCores,                 resource_family: standard,            location: leaseweb-dal13, unit_price: 0.0008000992, billed_by: duration, active_from: 2024-05-27T00:00:00Z }
```

Comment:
It's good to see no active billing records. However, we keep billing records indefinitely. What would happen if we wanted to calculate usage for these records again? It seems like a BillingRecord without a BillingRate is somewhat useless.